src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Book.ttf) format("truetype");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Heavy.ttf) format("truetype");
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  margin-top: 20px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.icon::placeholder {
+  font-family: inherit;
+  font-weight: 300;
+}
+
+.bigIcon {
+  font-family: "Avenir", georgia, serif;
+  font-size: 16px;
+  vertical-align: top;
+  height: 70px;
+  padding: 0 0 0 62px;
+  width: 100%;
+  margin-bottom: 20px;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  background: url(images/Search.svg) no-repeat 26px;
+  background-size: 19px 19px;
+  border-radius: 4px;
+}
+
+.bigIcon:hover {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  background: url(images/Search.svg) no-repeat 26px;
+  background-size: 19px 19px;
+}
+
+.bigIcon:focus {
+  font-family: "Avenir", georgia, serif;
+  font-weight: 800;
+  font-size: 16px;
+  vertical-align: top;
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: url(images/Search.svg) no-repeat 26px;
+  background-size: 19px 19px;
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.smallIcon {
+  font-family: "Avenir", georgia, serif;
+  font-size: 14px;
+  vertical-align: top;
+  height: 42px;
+  padding: 0 0 0 33px;
+  width: 100%;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  background: url(images/Search.svg) no-repeat;
+  background-position: left 12px top 14px;
+  background-size: 11px 11px;
+  border-radius: 4px;
+}
+
+.smallIcon:hover {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  background: url(images/Search.svg) no-repeat;
+  background-size: 11px 11px;
```

Comment:
no need to set it here at all
set in `:hover` (in all pseudo-classes) only things you want to change, it doesn't work properly:
![image](https://user-images.githubusercontent.com/86923089/226679012-1699ac06-2922-4695-9628-a99da87821d5.png)